doc/lms.mysql

```diff
@@ -3990,4 +3990,6 @@ INSERT INTO netdevicemodels (name, alternative_name, netdeviceproducerid) VALUES
 ('XR7', 'XR7 MINI PCI PCBA', 2),
 ('XR9', 'MINI PCI 600MW 900MHZ', 2);
 
+INSERT INTO taxes (value,taxed,label) VALUES (23, 1, 'VAT23');
```

Comment:
To mi się nie podoba, bo ja akurat lubię "23%" mieć jako etykietę.

Reply:
Nie martw się - nie będzie to miało wpływu na Twoją instancję.
Nadal będziesz tak miał :)

Reply:
Wyrzuć z PR kontrowersyjne zmiany to puścimy resztę.

Reply:
Czyli jeśli zmienię etykietę stawki podatku na "23%" to puścisz cały PR?

Reply:
W ogóle bez tego proszę, bo co firma to każdy inaczej etykietuje stawki VAT.

Reply:
Zgodnie z prośbą zostawiłem tylko te dwa commity w PR.